itest/lnd_route_blinding_test.go

```diff
@@ -1477,6 +1477,10 @@ func testBlindedPaymentHTLCReForward(ht *lntest.HarnessTest) {
 	ht.RestartNodeWithExtraArgs(
 		testCase.carol, []string{"--bitcoin.timelockdelta=18"},
 	)
+	// Nodes need to be connected otherwise the forwarding of the
```

Comment:
Style nit: should have a new line before the comment.

Reply:
Also re commits: you don't necessarily need to have the top-level commit message end in a period.

Reply:
had to redesign the fix, I think just ensuring they are not connected is not enough because in the meantime the htlc could have been replayed to the switch and failed, so we need to intercept the htlc here again => wait until the nodes are connected and then forward the htlc.